roles/wildfly_systemd/meta/argument_specs.yml

```diff
@@ -1,120 +1,175 @@
+                type: "bool"
+
+            # line 16 of defaults/main.yml
             wildfly_systemd_service_config_location:
                 default: "/usr/lib/systemd/system"
                 description: "Path for systemd unit file"
                 type: "str"
+
+            # line 17 of defaults/main.yml
             wildfly_systemd_service_config_file_suffix:
-                # line 10 of wildfly_systemd/defaults/main.yml
                 default: ".service"
                 description: "Systemd unit file extension"
                 type: "str"
+
+            # line 18 of defaults/main.yml
             wildfly_systemd_conf_file_suffix:
-                # line 11 of wildfly_systemd/defaults/main.yml
                 default: ".conf"
                 description: "Suffix for systemd conf file"
                 type: "str"
+
+            # line 19 of defaults/main.yml
             wildfly_systemd_service_config_file_template:
-                # line 12 of wildfly_systemd/defaults/main.yml
                 default: "templates/wfly.service.j2"
                 description: "Template for systemd unit"
                 type: "str"
+
+            # line 21 of defaults/main.yml
             wildfly_service_config_file_template:
-                # line 14 of wildfly_systemd/defaults/main.yml
                 default: "templates/wfly.conf.j2"
                 description: "Template for systemd config"
                 type: "str"
+
+            # line 22 of defaults/main.yml
             wildfly_service_config_file_location:
-                # line 15 of wildfly_systemd/defaults/main.yml
                 default: "/etc/"
                 description: "Path for wildfly systemd unit file"
                 type: "str"
+
+            # line 24 of defaults/main.yml
             wildfly_enable_yml_config:
-                # line 17 of wildfly_systemd/defaults/main.yml
                 default: false
                 description: "Enable yaml file configuration feature (WFCORE5343)"
                 type: "bool"
+
+            # line 25 of defaults/main.yml
             wildfly_yml_configs:
-                # line 18 of wildfly_systemd/defaults/main.yml
                 default: "[]"
                 description: "List of filenames for wildfly configuration bootstrap"
                 type: "list"
-            wildfly_instance_id:
-                # line 17 of wildfly_systemd/defaults/main.yml
-                required: false
-                description: "ID for instance when colocating EAP deployments"
-                type: "int"
-            wildfly_instance_name:
-                # line 17 of wildfly_systemd/defaults/main.yml
-                required: false
-                description: "Name for instance when colocating EAP deployments"
-                type: "str"
+
+            # line 26 of defaults/main.yml
+            wildfly_yml_configs_repository:
+                required: true
```

Comment:
this becomes `default: ""` without the required (if something needs to be specified then it needs a check in precond.yml)

Reply:
Fixed!